politeiawww/testing.go

```diff
@@ -245,11 +247,20 @@ func newTestPoliteiawww(t *testing.T) (*politeiawww, func()) {
 		userEmails:      make(map[string]uuid.UUID),
 		userPaywallPool: make(map[uuid.UUID]paywallPoolMember),
 		commentScores:   make(map[string]int64),
+		cmsDB:           testcmsdb.New(),
```

Comment:
This should only be set when in cmswww mode.  Move it down to the switch statement.